.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: macOS-10.14
```

Comment:
10.14 is not supported anymore: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources